 error[E0635]: unknown feature `proc_macro_span_shrink`
   --> .../host/share/cargo/registry/src/index.crates.io-*/proc-macro2-1.0.47/src/lib.rs:92:30
    |
 92 |     feature(proc_macro_span, proc_macro_span_shrink)
    |                              ^^^^^^^^^^^^^^^^^^^^^^
 
 error: could not compile `proc-macro2` (lib) due to previous error

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,3 +33,6 @@ rustc-demangle = "0.1.21"
 regex = "1.6"
 iced-x86 = "1.17"
 byteorder = "1.4.3"
+
+# rustc 1.72.0
+proc-macro2 = "1.0.66"
